crates/storage/src/params.rs

```diff
@@ -141,6 +144,13 @@ to_sql_builtin!(
     u8
 );
 
+impl ToSql for L1TransactionHash {
+    fn to_sql(&self) -> rusqlite::types::ToSqlOutput<'_> {
+        use rusqlite::types::{ToSqlOutput, ValueRef};
+        ToSqlOutput::Borrowed(ValueRef::Blob(self.as_bytes()))
+    }
+}
+
```

Comment:
Do we actually need this? Seems redundant now that we don't actually store these in storage.